slick/src/test/scala/geotrellis/slick/PostgisSpec.scala

```diff
@@ -52,147 +57,139 @@ class PostgisSpec extends FlatSpec with ShouldMatchers with TestDatabase {
   }
   val CityTable = TableQuery[City]
 
+  def createSchema() = db.run(CityTable.schema.create).futureValue
```

Comment:
Maybe a try/catch here for connection error and augment it with message to use `scripts/slickTestDB` for local tests.